eve/main.yml

```diff
@@ -311,6 +311,11 @@ stages:
     steps:
       - ShellCommand: *wait_for_docker
       - Git: *git_pull
+      - ShellCommand:
```

Comment:
The commit message describe what you do, and this we can see with the diff, especially since the commit is small (and that's good :+1: )

But it doesn't explain why we want to upload this magical file.